src/BuildScriptGenerator/DotNetCore/VersionProviders/DotNetCoreVersionProvider.cs

```diff
@@ -47,6 +53,8 @@ public string GetDefaultRuntimeVersion()
                     _onDiskVersionProvider.GetSupportedVersions();
             }
 
+            _logger.LogDebug("Got list of supported versions");
```

Comment:
nit: add `the` before `list`

Reply:
Since this is a very minor comment, I am going to handle it another PR (want to avoid the whole build to take place again).